main.js

```diff
@@ -1,57 +1,67 @@
 /* eslint-disable linebreak-style */
```

Comment:
- [ ] OPTIONAL
Project details on your _ReadMe_ do not match the current project. Kindly endeavor to update your ReadMe to correspond with the current project so as to make your _ReadMe_ professional.